page_analyzer/utils/db.py

```diff
@@ -89,7 +89,9 @@ def get_urls() -> List[Dict[str, Any]]:
     return actual_urls_data
 
 
-def create_check(url_data: Dict[str, Any]) -> bool:
+def create_check(url_data: Dict[str, Union[int, str]] | None) -> bool:
```

Comment:
In modern versions of Python ([since 3.10](https://docs.python.org/3/library/typing.html#typing.Union)) `Union` can be replaced with `|`

```suggestion
def create_check(url_data: Dict[str, int | str] | None) -> bool:
```